homeassistant/components/matter/api.py

```diff
@@ -33,7 +33,7 @@ async def _get_matter(
         hass: HomeAssistant, connection: ActiveConnection, msg: dict
     ) -> None:
         """Provide the Matter client to the function."""
-        matter: Matter = list(hass.data[DOMAIN].values())[0]
+        matter: MatterClient = next(iter(hass.data[DOMAIN].values()))
```

Comment:
It's the adapter that's stored in hass.data.

Reply:
yeah, typo